distribution/lib/Standard/Base/0.0.0-dev/src/Function.enso

```diff
@@ -88,7 +88,10 @@ flip f = (x -> y -> f y x)
    > Example
     IO.println <| [1, 2, 3].map (Function.const 7) # Prints '[7, 7, 7]'
 const : Any -> Any -> Any
-const x _ = x
+const x ~f =
+    black_hole ~_ = Nothing
```

Comment:
Maybe we should have a private helper method for this defined somewhere. `mark_used` or something like that.

Reply:
The problem is that `_` argument appear as Internal in the IDE so any public function should have named arguments.
The `_` (and `~_`) do work internally or for PRIVATE functions.